projects/network_tester/src/ui_task.c

```diff
@@ -888,6 +1029,36 @@ static void ui_print_mac_address_string(char* dest)
 {
     sprintf(dest,"$301$0-0-0-%09X", (unsigned int)sup_get_MAC_address());
 }
+
+static void ui_print_net_token(char *dest)
+{
+    if (token_edit_mode)
+    {
+        sprintf(dest, " NetToken: %" PRIx32, (long unsigned int)temp_token);
```

Comment:
This print won't work for network tokens with small values because it doesn't zero-pad (I realize this is my fault since I made this change when we were debugging). I think this line and the `sprintf` below should both be:

```
sprintf(dest, " NetToken: %08" PRIX32, net_token);
```